concent_api/core/tests/test_unit_authentication_utilities.py

```diff
@@ -115,14 +115,14 @@ def test_validate_golem_message_signed_with_key_should_not_raise_error_if_correc
         assert task_to_compute.SIGN is not False
 
         try:
-            validate_golem_message_signed_with_key(
-                task_to_compute,
+            handle_validating_if_list_of_golem_messages_is_signed_with_key(
                 CONCENT_PUBLIC_KEY,
+                task_to_compute,
             )
         except Http400:
             self.fail()
 
-    def test_validate_golem_message_signed_with_key_should_raise_error_if_incorrect_message_and_key_is_used(self):
+    def test_handle_validating_if_list_of_golem_messages_is_signed_with_key_should_raise_error_if_incorrect_message_and_key_is_used(self):
```

Comment:
please rename also the test, when `handle_validating_if_list_of_golem_messages_is_signed_with_key` function name is changed

Reply:
As above, lets decide.

Reply:
I would be for renaming